test/e2e/kuttl/ScaleUpAndDown/00-createCluster.yaml

```diff
@@ -3,7 +3,12 @@ kind: TestStep
 commands:
   - script: |
       BASEDIR=../../../..
-      VERSION=$(awk -F\" '/Version =/ {print $2}' $BASEDIR/version/version.go)-$CIRCLE_BRANCH
+      if [ $CIRCLE_TAG ]
+        then
+          VERSION="$(awk -F\" '/Version =/ {print $2}' $BASEDIR/version/version.go)-release"
+        else
+          VERSION="$(awk -F\" '/Version =/ {print $2}' $BASEDIR/version/version.go)-$CIRCLE_BRANCH"
+      fi
```

Comment:
@fdehay I would rather do 
```
VERSION=${CIRCLE_TAG:-CIRCLE_BRANCH}
```
I don't think we need to check the file at any point as we already use the right version in the tag